router/dataplane_spec.gobra

```diff
@@ -75,18 +82,40 @@ pred AccBfdSession(bfdSessions map[uint16]bfdSession) {
 
 pred AccForwardingMetrics(metrics map[uint16]forwardingMetrics) {
 	acc(metrics, 1/2) &&
-	forall fm forwardingMetrics :: fm in range(metrics) ==> acc(forwardingMetricsMem(fm), _)
+	forall id uint16 :: id in domain(metrics) ==> acc(forwardingMetricsMem(metrics[id], id), _)
```

Comment:
can we add a trigger here or will it be invalid?

Reply:
good point, done